lua/terrortown/lang/de.lua

```diff
@@ -1249,16 +1249,16 @@ L.hilite_win_innocents = "TEAM UNSCHULDIGE GEWANN"
 L.hilite_win_tie = "UNENTSCHIEDEN"
 L.hilite_win_time = "ZEIT VORBEI"
 
---L.tooltip_karma_gained = "Karma changes for this round:"
---L.tooltip_score_gained = "Score changes for this round:"
---L.tooltip_roles_time = "Role changes for this round:"
+L.tooltip_karma_gained = "Karma-Änderungen für diese Runde:"
+L.tooltip_score_gained = "Punkte-Änderungen für diese Runde:"
+L.tooltip_roles_time = "Rollen-Änderungen für diese Runde:"
```

Comment:
```suggestion
L.tooltip_roles_time = "Rollenänderungen für diese Runde:"
```

oder gar Rollenwechsel